lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
 
 __version__ = '0.2(2015-06-23)'
```

Comment:
This one is more for human level debugging for help calls. Until #3 is merged, this one might come in handy.